.circleci/real_config.yml

```
@@ -3168,7 +3168,9 @@ jobs:
                   curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && sudo install minikube-linux-amd64 /usr/local/bin/minikube
             - run:
                 name: Start defaultrm minikube
-                command: minikube start --profile defaultrm
+                command: |
+                  source tools/k8s/launch-minikube-with-gateway.sh defaultrm
+                  echo "export GATEWAY_IP=\"${GATEWAY_IP}\"" >> $BASH_ENV
```

Bash printf has a `%q` format which is perfect for general situation. It will take the value and quote it appropriately for a shell. You don't have to deal with any embedded quotes that way, but you do have to provide a newline. So, I guess it's a toss up whether or not this is preferable. Use it or not; this is more of an FYI than anything, since an IP doesn't need quotes either way. :)
```suggestion
                  printf "export GATEWAY_IP=%q\n" "$GATEWAY_IP" >> $BASH_ENV
```